NodeBase/entry_point.sh

```diff
@@ -25,6 +25,10 @@ if [ ! -z "$REMOTE_HOST" ]; then
   REMOTE_HOST_PARAM="-remoteHost $REMOTE_HOST"
 fi
 
+if [ ! -z "$FIREFOX_VERSION" ]; then
+  sudo ln -fs /opt/firefox/${FIREFOX_VERSION}/firefox /usr/bin/firefox
+fi
+
```

Comment:
so no matter what time of node you are using, firefox will be in /usr/bin/firefox?  say i'm using NodeChrome, /usr/bin/firefox will exist?